submissions/UA-girl/Task-HTML-CSS/index.html

```diff
@@ -0,0 +1,1538 @@
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Kyiv"
+                  name="city"
+                  checked
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Київ</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Lviv"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Львів</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Odesa"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Одеса</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Dnipro"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Дніпро</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Kharkiv"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Харків</span>
+              </label>
+              <label class="booking-label">
+                <input
+                  type="radio"
+                  class="booking-input booking-input__city visually-hidden"
+                  value="Kharkiv"
+                  name="city"
+                />
+                <span class="booking-ico booking-ico__city"></span>
+                <span class="booking-name booking-city__name">Полтава</span>
+              </label>
+            </fieldset>
+            <div class="booking-details">
+              <span class="booking-ico booking-ico__place">1</span>
+              <span>Вільне місце</span>
+              <span
+                class="booking-ico booking-ico__place booking-ico__place__disabled"
+                >1</span
+              >
+              <span>Недоступне</span>
+            </div>
```

Comment:
Reduce space between text and "icon" for better understanding

![image](https://user-images.githubusercontent.com/16196199/106382073-c208dd00-63c5-11eb-8e7a-96ebced6d77a.png)

Reply:
Fixed